src/Models/Nonhydrostatic2DModels/nonhydrostatic_2d_model.jl

```diff
@@ -140,3 +140,10 @@ function Models.make_ode_function(model::Nonhydrostatic2DModel{FT}) where {FT}
         return dY
     end
 end
+
+function get_velocities(u, model::Nonhydrostatic2DModel)
```

Comment:
I suggest use `model_fields` instead of `u` here because it is easily confused with the zonal velocity.